libraries/I2S/src/I2S.cpp

```diff
@@ -363,6 +362,9 @@ size_t I2S::write(int32_t val, bool sync) {
     if (!_running || !_isOutput) {
         return 0;
     }
+    //Serial.print("pio_sm_is_tx_fifo_empty(_pio,_sm): ");
```

Comment:
Can we blow this leftover debug away, please?

Reply:
oh yes of course. Sorry, I missed that